workspace/workspace.go

```diff
@@ -57,6 +57,16 @@ func (ws Workspace) PotentialExercises() ([]Exercise, error) {
 			continue
 		}
 
+		if topInfo.Name() == "teams" {
```

Comment:
I needed to add support for teams to `PotentialExercises` as before this change, it doesn't find them

Reply:
Would you mind extracting this change into a separate PR? (We should probably include tests for the change).